releasenotes-builder/README.md

```diff
@@ -111,6 +111,8 @@ Generated files will be at specified output location.
 
 **sfRssProperties** - (optional) path to a properties file for publication to RSS.
 
+**validateVersion** - (optional) validate pom version in issues labels.
```

Comment:
To be honest, I am not a big fan of `validateVersion` since we are not only validating pom version but also release notes labels. `validateVersionWithReleaseNotesLabels` would sound more appropriate. But these are just my thoughts.

Reply:
Alright, for internal use makes sense.